apps/static/dance/ai/model/cachedbackgroundmap.json

```diff
@@ -0,0 +1,165 @@
+{

```

Comment:
Why are there two separate sets of json files?

Reply:
These were older JSONs for our in-house emojis while we were testing with them; we can delete them now if we don't want them anymore since we don't actively use them